pkg/apisix/cluster.go

```diff
@@ -750,10 +750,16 @@ func (c *cluster) getList(ctx context.Context, url, resource string) ([]string,
 		return nil, err
 	}
 
-	var listResponse []string
+	var listResponse map[string]interface{}
 	dec := json.NewDecoder(resp.Body)
 	if err := dec.Decode(&listResponse); err != nil {
 		return nil, err
 	}
-	return listResponse, nil
+	var res []string = make([]string, len(listResponse))
```

Comment:
```suggestion
	res []string := make([]string, 0, len(listResponse))
```

After that, just using `append`, we don't need the indice `i`.

Reply:
Done